aggregator/src/aggregation/decoder/witgen/types.rs

```diff
@@ -539,6 +545,140 @@ pub struct BitstreamReadRow {
     pub is_zero_bit_read: bool,
 }
 
+/// Sequence data is interleaved with 6 bitstreams. Each producing a different type of value.
+#[derive(Clone, Copy, Debug)]
+pub enum SequenceDataTag {
+    NULL = 0,
+    LiteralLength_FSE,
+    MatchLength_FSE,
+    CookedMatchOffset_FSE,
+    LiteralLength_Value,
+    MatchLength_Value,
+    CookedMatchOffset_Value,
+}
+
+/// A single row in the Address table.
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct AddressTableRow {
+    /// Whether this row is padding for positional alignment with input
+    pub s_padding: u64,
+    /// Instruction Index
+    pub instruction_idx: u64,
+    /// Literal Length (directly decoded from sequence bitstream)
+    pub literal_length: u64,
+    /// Cooked Match Offset (directly decoded from sequence bitstream)
+    pub cooked_match_offset: u64,
+    /// Match Length (directly decoded from sequence bitstream)
+    pub match_length: u64,
+    /// Accumulation of literal length
+    pub literal_length_acc: u64,
+    /// Repeated offset 1
+    pub repeated_offset1: u64,
+    /// Repeated offset 2
+    pub repeated_offset2: u64,
+    /// Repeated offset 3
+    pub repeated_offset3: u64,
+    /// The actual match offset derived from cooked match offset
+    pub actual_offset: u64,
+}
+
+/// Data for BL and Number of Bits for a state in LLT, CMOT and MLT
+#[derive(Clone, Debug)]
+pub struct SequenceFixedStateActionTable {
```

Comment:
Upstream we have already pushed a fixed table for these 3. May be we can re-use them?

Refer here: https://github.com/scroll-tech/zkevm-circuits/blob/feat/da-compression/aggregator/src/aggregation/decoder/tables/rom_sequence_codes.rs